app/services/csv/sales_log_csv_service.rb

```diff
@@ -74,7 +74,7 @@ def value(attribute, log)
         value = log.send(attribute)
         get_label(value, attribute, log)
       elsif DATE_FIELDS.include? attribute
-        log.send(attribute)&.iso8601
```

Comment:
I'm not sure of why this wasn't already public_send given the DATE_FIELDS are only created_at and updated_at, both of which are accessible by public_send so I've changed this

Reply:
Wouldn't we want to show the time of created at tho?

Reply:
Ah great point, I thought it didn't specify in the ticket but it actually does - users are having issues with excel when there are timestamps but doesn't seem like it's an issue for created/updated_at, will change